### DISCUSSION TITLE
Input Shaping Sovol Klipper Screen

### DISCUSSION THREAD

**Discussion by troymroberts** [bassamanator/everything-sovol-sv06#43] [Q&A]
Hey everyone, I'm sure I'm missing something totally obvious. So I installed the sovol klipper screen and updated the firmware and the klipper from the instructions in the repo.

However I'm not finding an option to enable input shaping with the supplied accelerometer.

Any assistance would be appreciated

**Comment by bassamanator** [comment]
Don't own the screen unfortunately. I would check on youtube, or the manual?
